code/game/area/Space_Station_13_areas.dm

```diff
@@ -1025,6 +1025,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Brig"
 	icon_state = "brig"
 	mood_bonus = -3
+	mood_job_allowed = list(JOB_NAME_HEADOFSECURITY,JOB_NAME_WARDEN)
```

Comment:
Why aren't sec officers, detective and brig phys included?

Reply:
warden is the only one who derives joy from these steel bars, and the Hos only sees JUSTICE!

Reply:
secoffs have to go in there for their job, exempt them